### ISSUE TITLE
uninitialized constant WillPaginate::ViewHelpers with will_paginate v.3.0.0

### ISSUE THREAD

**Issue by dima4p** [leikind/wice_grid#57]
After I've upgraded will_paginate to the latest version (v.3.0.0) I get:
carrierwave/ruby/1.9.1/gems/wice_grid-3.0.2/lib/wice_grid.rb:35:in `block in class:WiceGridRailtie': uninitialized constant WillPaginate::ViewHelpers (NameError)

I tried to figure why but have not enough time now for the investigation to finish. I hope you will find quickly the reason.
It seems the require 'will_paginate/view_helpers' is needed.

**Comment by dima4p** [comment]
with 3.0.pre2 works, with 3.0.pre4 does not

**Comment by leikind** [comment]
Line 35 in wice_grid is 

```
 Wice::GridRenderer.send(:include, ::WillPaginate::ViewHelpers)
```

Apparently WillPaginate::ViewHelpers exists no more in will_paginate. I suggest you stay on  3.0.pre2  until I figure this out

**Comment by dima4p** [comment]
less lib/will_paginate/view_helpers.rb

**Comment by leikind** [comment]
fixed in 3.0.3

**Comment by dima4p** [comment]
Спасибо!
